pkg/sdkserver/sdkserver_test.go

```diff
@@ -753,19 +719,12 @@ func TestSDKServerWatchGameServerFeatureSDKWatchSendOnExecute(t *testing.T) {
 		}
 	}
 
-	// if SDKWatchSendOnExecute feature is turned on, there are two stream.Send() calls should happen:
```

Comment:
nit: Instead of removing the whole comment, please change it to:

```
// There are two stream.Send() calls should happen: one in sendGameServerUpdate, another one in WatchGameServer.
```

(with an appropriate line break).